packages/react-components/react-nav-preview/src/components/NavCategoryItem/useNavCategoryItem.tsx

```diff
@@ -42,6 +42,9 @@ export const useNavCategoryItem_unstable = (
     root: slot.always(
       getIntrinsicElementProps('button', {
         ref,
+        role: 'menuitem',
+        'aria-level': 1,
```

Comment:
`aria-level` actually isn't supported on menus, only tree, treegrid, and heading (and comment sometimes, that one's more obscure).